.actor/input_schema.json

```diff
@@ -136,11 +136,14 @@
             "description": "If enabled, the Actor attempts to close or remove cookie consent dialogs to improve the quality of extracted text. Note that this setting increases the latency.",
             "default": true
         },
-        "useCheerioCrawler": {
-            "title": "Use Cheerio Crawler",
-            "type": "boolean",
-            "description": "If enabled, the Actor uses the Cheerio Crawler to extract the target web page content.",
-            "default": false
+        "scrapingTool": {
+            "title": "Which scraping tool to use",
+            "type": "string",
+            "description": "Choose what scraping tool to use for extracting the target web pages. The Browser tool is more powerful and can handle JavaScript heavy websites. While the Plain HTML tool is about two times faster.",
+            "editor": "select",
+            "default": "playwright",
+            "enum": ["playwright", "cheerio"],
+            "enumTitles": ["Browser", "Plain HTML"]
```

Comment:
I meant `Plain HTTP` but honestly not sure what is the best term. WCC uses [Raw HTTP](https://apify.com/apify/website-content-crawler/input-schema) so maybe let's use that too, we have same/similar users.